<?xml version="1.0" encoding="UTF-8"?>
<sequence name="com.wso2telco.dep.hub.paymentapi.check.limit.Sequence" onError="com.wso2telco.dep.hub.paymentapi.check.limit.error.Sequence" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
    <property expression="get-property('CONSUMER_KEY')"
              name="consumerKey" scope="default" type="STRING"/>
    <property expression="get-property('endUserId')" name="endUserId"
              scope="default" type="STRING"/>
    <class name="com.wso2telco.dep.spend.limit.mediation.mediator.LimitCheckMediator"/>

    <!--since the checklimit is called from the charge handler, on success from here it should return to that sequence-->
    <!--<sequence key="chargeRequestSeq"/>-->
</sequence>

